db/cardano-sl-db.cabal

```diff
@@ -141,6 +141,7 @@ library
                      , ekg-core
                      , ether >= 0.5
                      , exceptions
+                     , extra
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/cardano-sl-db.cabal#L144